prediction_market_agent/agents/logprobs_agent/deploy.py

```
@@ -0,0 +1,261 @@
+
+* Return response in following format:
+* First return your rationale of why did you decided given way in "rationale": "YOUR RATIONALE HERE",\n
+* Then return your positive prediction percentate inside of field "p_yes": "YOUR PREDICTION  IN RANGE [0-1]"\n
+
+* Your output response must be only a single JSON object to be parsed by Python's "json.loads()".
+* RETURN ONLY RESPONSE CONTAINING ONLY FOLLOWING NOTHING ELSE  "rationale": "YOUR RATIONALE HERE",\n "p_yes": "YOUR PREDICTION  IN RANGE [0-1]",\n
+* Output only the JSON object in your response. Do not include any other contents in your response.
+
+"""
+
+class DeployableLogProbsAgent(LogProbsAgent):
```

Ruff (0.11.9)

Undefined name `LogProbsAgent`

(F821)